sql-plugin/src/main/311until320-all/scala/org/apache/spark/storage/RapidsShuffleBlockFetcherIterator.scala

```diff
@@ -835,6 +838,7 @@ private class RapidsBufferReleasingInputStream(
     }
   }
 
+  @scala.annotation.nowarn("msg=method closeQuietly in class IOUtils is deprecated")
```

Comment:
I'd rather see these put up as separate PRs in the future so they can be tracked properly.  Its fine for now.